docker_explorer/de.py

```diff
@@ -52,11 +52,8 @@ def _SetDockerDirectory(self, docker_path):
     """
     self.docker_directory = docker_path
     if not os.path.isdir(self.docker_directory):
-      err_message = (
-          '{0:s} is not a Docker directory\n'
-          'Please specify the Docker\'s directory path.\n'
-          'hint: de.py -r /var/lib/docker').format(self.docker_directory)
-      raise errors.BadStorageException(err_message)
+      msg = '{0:s} is not a Docker directory\n'.format(self.docker_directory)
```

Comment:
Just FYI, there's a `\n` here which is absent from the other exceptions.

Reply:
done